docs/API.rst

```diff
@@ -4,3 +4,21 @@ API Reference
 
 .. automodule:: loky
     :members: get_reusable_executor
+
+
+Task & results serialization
+----------------------------
+
+To share function definition across multiple python processes, it is necessary to rely on a serialization protocol. The standard protocol in python is :mod:`pickle` but this protocol has several limitation. For instance, it cannot serialize functions which are defined interactively or in the :code:`__main__` module.
```

Comment:
Actually, cloudpickle respects the pickle protocol: you can use `pickle.load` to unpickle something generated by `cloudpickle.dump`

So instead I would write:

The standard protocol in python is :mod:`pickle` but its default implementation in the standard library has several limitations.